File: rafael/arrays/SequenciaAleatoria.java
class SequenciaAleatoria {
	public static void main(String[] args) {
		int[] array = new int[10];

		for (int i = 0; i < array.length; i++) {
			array[i] = (int)(Math.random() * 100);
		}

		for (int i = 0; i < array.length; i++) {
			System.out.println(array[i]);
		}
	}
}

File: rafael/arrays/SequenciaAleatoria.cs
class SequenciaAleatoria 
{
	static void Main() 
	{
		System.Random gerador = new System.Random();

		int[] array = new int[10];

		for (int i = 0; i < array.Length; i++) 
		{
			array[i] = (int)(gerador.NextDouble() * 100);
		}

		for (int i = 0; i < array.Length; i++) 
		{
			System.Console.WriteLine(array[i]);
		}
	}
}
